README.md

```diff
@@ -53,8 +81,9 @@ $ make build-c # builds our c example
 $ make run-c # runs our c example
 ```
 
-> Note: There are also Rust examples that can be built (`build-rust`) and ran (`run-rust`). However, some of these examples have some dependencies on environment variables or local programs (e.g., `etcd`), so it probably won't work right off the gate like the C one did.
+# Building Rust Examples
 
+There are also Rust examples that can be built (`build-rust`) and ran (`run-rust`). However, some of these examples have some dependencies on environment variables or local programs (e.g., `etcd`), so it probably won't work right off the gate like the C one did.
```

Comment:
Maybe just tell them that you can build them but you don't recommend running because it requires additional setup that we don't have documented yet (or plan to never document)?

Reply:
Thank you for your review, Carolyn!